incubator/jaeger/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 1.9.0
+appVersion: 1.10.0
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
-version: 0.8.3
+version: 0.9.0
```

it might be a breaking change and also since the chart is in stable lets set to 1.0.0 
can you also update the docs?
```suggestion
version: 1.0.0
```

---

Thanks for taking a look @cpanato !
@desaintmartin also pointed these issues out.
With regards to breaking changes, I've created an `upgrading` section and added a note about it.

With regards to `1.0.0` it is still in the incubator repo, my understanding is that `1.0.0` and beyond is when the chart is moved to the stable repo. Is this a possibility?

---

sorry! i misread :( ignore my comment about the version

---

haha  no worries! :)